README.md

```diff
@@ -31,15 +31,33 @@ Old working document of the [charter](https://docs.google.com/document/d/1w_lo2R
 
 <!-- https://contribute.cncf.io/maintainers/governance/charter/ -->
 
-The CNCF Artifacts Working Group is under the App Delivery Technical Advisory Group hosted by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).
-
-`https://github.com/cncf/tag-app-delivery/tree/main/artifacts-wg`
+The CNCF Artifacts Working Group is under the [App Delivery Technical Advisory Group](https://github.com/cncf/tag-app-delivery/tree/main/artifacts-wg) hosted
+by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).
 
 ## Chairs
 
 - Alex Flom (@afflom)
```

Comment:
Should this be updated based on https://github.com/cncf/tag-app-delivery/issues/516?

Reply:
Made this a separate PR
https://github.com/cncf-tags/wg-artifacts/pull/32